docs/user-guide/api-mediation/configuration-at-tls.md

```diff
@@ -1,23 +1,254 @@
+components.*.certificate.keystore.alias: <certificate alias / label from AT-TLS rule>
+```
+
+If there is an outbound AT-TLS rule configured for the link between the API Gateway and z/OSMF, update or set the `zowe.zOSMF.scheme` to `http`.
+
+:::note
+AT-TLS is not yet supported in the API Cloud Gateway Mediation Layer component.
+:::
+
+### Other Zowe components
+
+Given the central characteristic of the API Mediation Layer Gateway, other components that need to interact with it need to be configured
+
+An example of this is the Zowe ZLUX App Server.
+
+## Security considerations
+
+Configuring AT-TLS for the Zowe API Mediation Layer needs careful consideration regarding the security settings, especially around the Client Certificate authentication feature in the Zowe API Mediation Layer components as well as for the Onboarded services that support the x.509 client certificates authentication scheme.
+
+In general terms, the outbound AT-TLS rules (i.e. to make a transparent https call through http) that are configured to send the server certificate should be limited to the services that __require__ service to service authentication, such as the case of the API Gateway authenticating with the Discovery Service.
+
+**Note:** The Discovery Service endpoints are not reachable by standard API Gateway routing by default.
+
+## AT-TLS rules
+
+This section describes the suggested AT-TLS settings. It is meant as a guideline and helps showcase important aspects.
+
+### Inbound rules
+
+A generic inbound rule can be set for all of the core API Mediation Layer services:
+
+```pagent
+TTLSRule ApimlServerRule
+{
+  LocalAddr All
+  RemoteAddr All
+  LocalPortRange 10310-10320
+  Jobname ZWE*
+  Direction Inbound
+  TTLSGroupActionRef ServerGroupAction
+  TTLSEnvironmentActionRef ApimlServerEnvironmentAction
+  TTLSConnectionActionRef ApimlServerConnectionAction
+}
+
+TTLSGroupAction ServerGroupAction
+{
+  TTLSEnabled On
+}
+
+TTLSEnvironmentAction ApimlServerEnvironmentAction
+{
+  HandshakeRole ServerWithClientAuth
+  EnvironmentUserInstance 0
+  TTLSEnvironmentAdvancedParmsRef ServerEnvironmentAdvParms
+  TTLSKeyringParmsRef ApimlKeyring
+}
+
+TTLSConnectionAction ApimlServerConnectionAction
+{
+  HandshakeRole ServerWithClientAuth
+  TTLSCipherParmsRef CipherParms
+  TTLSConnectionAdvancedParmsRef ApimlConnectionAdvParms
+}
 ```
-components.*.spring.profiles.active=attls
-components.*.server.ssl.enabled=false
-components.*.server.internal.ssl.enabled=false
+
+The `PortRange` of this inbound rule is taken from the list of API Mediation Layer components in the `zowe.yaml` file, it should cover the following components:
+
+- Gateway: default port 7554
+- Discovery: default port 7553
+- Caching Service: 7555
+- API Catalog: default port 7552
+- Metrics Service: default port 7551
+
+Replace `ApimlKeyring` with the one configured for your installation. Follow [these instructions]() to configure Keyrings for your Zowe instace.
```

Comment:
Link to the "instructions"?

Reply:
I'm looking for the main article about keyring configurations we have